packages/containers/src/Notification/Notification.constant.js

```diff
@@ -0,0 +1,3 @@
+export default {
+	PUSH_NOTIFICATION: 'NOTIFICATION__PUSH_NOTIFICATION',
```

Comment:
Other containers does not prefix actions by the container name. Could you remove that please ?